oval:
  tag_id: 58
  ab_or: 0.065

circle:
  blabla: 4.68
  tag_id: 58

main:
  blabla: 4.68
  tag_id: 58